tests/test_base.py

```diff
@@ -0,0 +1,18 @@
+import unittest
+from app.start import server, db
+
+class TestBase:
```

Comment:
Why a nested class? Just put both classes at module-level

Reply:
It's not necessary but this way they won't be called on by test discovery according to SO